boilerplate/app/components/Icon.tsx

```diff
@@ -63,7 +68,7 @@ export function Icon(props: IconProps) {
       <Image
         style={[
           $imageStyle,
-          { tintColor: color },
+          color && { tintColor: color },
```

Comment:
i don't think we necessarily needs this. if color is undefined, well it's undefined. i'm being nitpicky though :)

Reply:
Will this overwrite `$imageStyle`'s tintColor though?

Reply:
I mostly did it so it would match how `size` was handled one line down. Also, if someone defined a default `tintColor` in the `$imageStyle`, not having this guard would cause a bug.

Reply:
Yes it will. But $imageStyle doesn't have a tintColor (the default tintColor is the exported icon color since sometimes icons will have multiple colors: e.g. ladybug).